### DISCUSSION TITLE
docker可以使用web端口加465 995创建邮件服务

### DISCUSSION THREAD

**Discussion by nyamuch1** [Jinnrry/PMail#201] [General]
![image](https://github.com/user-attachments/assets/957978ff-2343-4d98-a056-935dead438e2)
需手动配置证书

**Comment by Jinnrry** [comment]
what？？？没看懂你说啥

**Reply by nyamuch1** [reply]
就是说，可以跳过25端口110端口配置，只需要465和995。还能用微软的邮件应用连接邮箱
![image](https://github.com/user-attachments/assets/e48e4bb6-ae9d-4c77-aa33-9ab65f760077)

**Reply by Jinnrry** [reply]
25端口是必须的，没有25端口没法收发邮件。465 995只能解决你客户端能连上，但是别人发件给你和你发件给别人，必须走25端口

**Reply by nyamuch1** [reply]
我也困惑了，之前是能收发的，还有垃圾邮件，好像更新开pop之后收不了了，但是能发。
![image](https://github.com/user-attachments/assets/343b0ff0-772d-4a84-9ca8-12da03783371)
难搞

**Reply by Jinnrry** [reply]
因为有少量服务商支持使用SMTPS发件，比如PMail本身默认就是SMTPS优先，对方不支持的情况下才降级使用SMTP。垃圾邮件的话对方为了提高收件率，肯定什么协议都来一遍呗。

但是目前，市面上绝大部分服务商，服务端通信都是不支持SMTPS的，这个我以前是专门测试过的。因此我才说25端口是必须的

**Reply by nyamuch1** [reply]
![image](https://github.com/user-attachments/assets/7a3016c1-acfc-4e44-a965-8a95cc871be6)
![image](https://github.com/user-attachments/assets/8c9c986c-e20e-44bc-bafd-6b46fdc59a9a)
![image](https://github.com/user-attachments/assets/5a680797-4b84-40a1-acbc-f841335a652d)
确实能收到，应该就是没走25端口？算是差中取优吧

**Reply by nyamuch1** [reply]
你的想法是怕有些服务商不支持465端口的话，可能会让我们错过一些可能重要但是又收不到邮箱吗

**Reply by nyamuch1** [reply]
想了想，应该是走POP3了，看端口情况110和995端口都是开着的？不确定，不是很懂

**Comment by nyamuch1** [comment]
我上面那张图片就是用qq邮箱发送到自建邮箱的记录。我挂一会儿看看还能不能收到垃圾邮件



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年9月12日(周四) 下午3:34
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Jinnrry/PMail] docker可以使用web端口加465 995创建邮件服务 (Discussion #201)




 
因为有少量服务商支持使用SMTPS发件，比如PMail本身默认就是SMTPS优先，对方不支持的情况下才降级使用SMTP。垃圾邮件的话对方为了提高收件率，肯定什么协议都来一遍呗。
 
但是目前，市面上绝大部分服务商，服务端通信都是不支持SMTPS的，这个我以前是专门测试过的。因此我才说25端口是必须的
